python/requirements/ml/tune-requirements.txt

```diff
@@ -7,6 +7,7 @@ bayesian-optimization==1.4.3
 ConfigSpace==0.7.1; python_version < "3.12"
 hpbandster==0.7.4; python_version < "3.12"
 
-hyperopt==0.2.7
+hyperopt @ git+https://github.com/hyperopt/hyperopt.git@2504ee61419737e814e2dec2961b15d12775529c
```

Comment:
need to use this version from the master branch because its later release (0.2.7) doesn't work with python 3.12

Reply:
don't we already skip all these tests? maybe we should remove the dep for now then

Reply:
ah good catch, i wasn't intended to skip hyperopt but then added to a few; will update ;)